allennlp/commands/predict.py

```diff
@@ -107,13 +93,9 @@ def _get_predictor(args: argparse.Namespace) -> Predictor:
     if args.predictor:
         # Predictor explicitly specified, so use it
         return Predictor.from_archive(archive, args.predictor)
-
-    # Otherwise, use the mapping
-    model_type = archive.config.get("model").get("type")
-    if model_type not in DEFAULT_PREDICTORS:
-        raise ConfigurationError(f"No known predictor for model type {model_type}.\n"
-                                 f"Specify one with the --predictor flag.")
-    return Predictor.from_archive(archive, DEFAULT_PREDICTORS[model_type])
+    else:
+        # Otherwise, use the default
+        return Predictor.from_archive(archive)
```

Comment:
do you even need the `if` here? (if args.predictor is not specified it will be `None`? which is the default value for that argument anyway)

Reply:
Good call, thanks.